Gemfile

```diff
@@ -46,6 +46,7 @@ gem "htauth",                         "2.0.0",         :require => false
 gem "inifile",                        "~>3.0",         :require => false
 gem "jbuilder",                       "~>2.5.0" # For the REST API
 gem "manageiq-api-client",            "~>0.1.0",       :require => false
+gem "more_core_extensions",           "~>3.1"
```

Comment:
Please keep the Gemfile sorted alphabetically, it makes finding dependencies easier.